experimental/packages/opentelemetry-browser-detector/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/opentelemetry-browser-detector",
+  "version": "0.32.0",
+  "description": "OpenTelemetry Resource Detector for Browser",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "version": "node ../../../scripts/version-update.js",
+    "watch": "tsc -w",
+    "precompile": "lerna run version --scope $(npm pkg get name) --include-dependencies",
+    "prewatch": "npm run precompile",
+    "peer-api-check": "node ../../../scripts/peer-api-check.js",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../"
+  },
+  "keywords": [
+    "opentelemetry",
+    "browser",
+    "resource",
+    "detector"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@babel/core": "7.16.0",
+    "@opentelemetry/api": "^1.0.0",
+    "@types/mocha": "9.1.1",
+    "@types/node": "18.6.5",
+    "@types/sinon": "10.0.13",
+    "codecov": "3.8.3",
+    "cpx": "1.5.0",
+    "mocha": "10.0.0",
+    "nyc": "15.1.0",
+    "rimraf": "3.0.2",
```

Comment:
Ditto

Reply:
most of these are used in the scripts(in package.json) for testing. I see them used in other packages too. I am not sure if they are a requirement for every package. Can you check and tell me if it is also good to remove the scripts. Also can you tell me if there is an additional work to publish this as a package when approved and merged ?

Reply:
These dependencies are not required. If they are not used in this package, they can be removed.

Reply:
removed